### PR TITLE
common: account rewards when resolving unallocate action delta

### LINKED ISSUE #620: Feasibility check should consider available Rewards
The current stake feasibility check should also consider the amount of available rewards in the balance.

### PR DESCRIPTION
**Pull request by tilacog**
Fixes #620.

### CODE REVIEW

**Comment by tilacog** [comment]
I suspect we might be being rate-limited, and this could be causing some tests to fail.

File: packages/indexer-common/src/indexer-management/allocations.ts [hunk 1]
```diff
@@ -1131,27 +1133,36 @@ export class AllocationManager {
...
       }
+
       // Fetch the allocation on chain to inspect its amount
       const allocation = await this.networkMonitor.allocation(action.allocationID)
 
-      /* We intentionally don't check if the allocation is active now because it will be checked
-       * later when we prepare the transaction */
+      // Accrue rewards, except for null or zeroed POI
+      rewards =
+        !action.poi || action.poi === '0' || action.poi === '0x0'
```
Comment (fordN) [inline packages/indexer-common/src/indexer-management/allocations.ts:1153]: When comparing against a "zero" POI I'd suggest using `hexlify()` to generate the comparison value. 

Like: `action.poi === utils.hexlify(Array(32).fill(0))`

**Review by fordN — CHANGES_REQUESTED** [review]

File: packages/indexer-common/src/indexer-management/allocations.ts (hunk 1 above)
Comment (tilacog) [reply packages/indexer-common/src/indexer-management/allocations.ts]: Fixed in c8b6254

**Review by fordN — APPROVED** [review]
👍

**Comment by tilacog** [comment]
Force pushed for rebasing

**Pull request merged**